### PR TITLE
MutatorMath input

### PR DESCRIPTION
**Pull request by jamesgk**
cc @brawer

### CODE REVIEW

**Comment by dougfelt** [comment]
LGTM

File: Lib/fontmake/font_project.py [hunk 1]
```diff
@@ -25,19 +25,12 @@
 from cu2qu.rf import fonts_to_quadratic
 from fontTools import subset
 from glyphs2ufo.glyphslib import build_masters, build_instances
+from mutatorMath.ufo import build as build_designspace
+from robofab.world import OpenFont
```
Comment (anthrotype) [inline Lib/fontmake/font_project.py]: hmm. why do you [drop the robofab dependency in ufoLib](https://github.com/googlei18n/ufo2ft/pull/31) while at the same time you require it in fontmake?

Comment (anthrotype) [inline Lib/fontmake/font_project.py]: sorry I just noticed the commit message:

> For the time being cu2qu only works with robofab, so don't try using defcon.

well, we shall fix this in cu2qu then. Once the pens are merged in cu2qu, defcon should work.

Comment (adrientetar) [inline Lib/fontmake/font_project.py]: > Once the pens are merged in cu2qu, defcon should work.

Yeah but the FontCollection stuff that generates compatible fonts in parallel won't work with a basic pen, will it?

Comment (anthrotype) [inline Lib/fontmake/font_project.py]: no, not yet. If have some ideas as to how to adapt the pen protocol to the *Collection classes in `cu2qu.rf` module, but don't have anything to show yet. Suggestions are welcome.

**Pull request merged**